### PR TITLE
Validate eve.json output /v5

### PR DESCRIPTION
**Pull request by TharushiJay**
Validates eve.json output using JSON schema.

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/1369

Changes from last PR : #440

- Added `--quiet `to suppress all output
- Resolved path errors
- Threw error in `run.py` if `check-eve.py` exited with error
- Broke commit into 3

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -872,6 +872,9 @@ def run_test(dirpath, args, cwd, suricata_config):
         elif results["success"] > 0:
             with lock:
                 count_dict["passed"] += 1
+        check_output = subprocess.call(["../suricata-verify/check-eve.py", outdir, "-q"])
```
Comment (jasonish) [inline run.py:875]: We need to use TOPDIR here to get the path correct.  I can suricata-verify like `../../suricata-verify.run.py` from my Suricata source directory, and the path is not found.

File: run.py [hunk 2]
```diff
@@ -872,6 +872,9 @@ def run_test(dirpath, args, cwd, suricata_config):
...
+        if check_output == 0:
+            raise TestError("Invalid JSON schema")
```
Comment (jasonish) [inline run.py:877]: This should be `!= 0`.

**Review by jasonish — CHANGES_REQUESTED** [review]
The output is still a little odd when the schema is updated to fail:
```
===> alert-distance-within-1: OK
===> alert-distance-within-1: FAILED: Invalid JSON schema
===> alert-no-3whs-established: OK
===> alert-no-3whs-established: FAILED: Invalid JSON schema
===> alert-testmyids: OK
===> alert-testmyids: FAILED: Invalid JSON schema
```
but lets focus on the other issues first.

**Comment by TharushiJay** [comment]
Changes continued in #464

**Pull request closed without merging**